version.properties

```diff
@@ -1,4 +1,4 @@
 #Wed Dec 07 09:39:06 UTC 2022
-dependency.coreSdk.version=0.26.5
+dependency.coreSdk.version=0.27.0
```

Comment:
`0.27.0` doesn't exist yet, does it?

Reply:
it shouldn’t be there, I changed it for test purposes and forgot to change it back.

Reply:
fixed [81eeee9](https://github.com/salemove/android-sdk-widgets/pull/418/commits/81eeee9b0c65adf94e0f404c3c89a4ec23414ef7)